sdk/cosmos/azure-cosmos-spark_3-3_2-12/README.md

```diff
@@ -0,0 +1,146 @@
+| 4.8.0        | 3.1.1 - 3.1.2            | 8                    | 2.12                     | 8.\*, 9.\*                    |
+| 4.7.0        | 3.1.1 - 3.1.2            | 8                    | 2.12                     | 8.\*, 9.\*                    |
+| 4.6.2        | 3.1.1 - 3.1.2            | 8                    | 2.12                     | 8.\*, 9.\*                    |
+| 4.6.1        | 3.1.1 - 3.1.2            | 8                    | 2.12                     | 8.\*, 9.\*                    |
+| 4.6.0        | 3.1.1 - 3.1.2            | 8                    | 2.12                     | 8.\*, 9.\*                    |
+| 4.5.3        | 3.1.1 - 3.1.2            | 8                    | 2.12                     | 8.\*, 9.\*                    |
+| 4.5.2        | 3.1.1 - 3.1.2            | 8                    | 2.12                     | 8.\*, 9.\*                    |
+| 4.5.1        | 3.1.1 - 3.1.2            | 8                    | 2.12                     | 8.\*, 9.\*                    |
+| 4.5.0        | 3.1.1 - 3.1.2            | 8                    | 2.12                     | 8.\*, 9.\*                    |
+| 4.4.2        | 3.1.1 - 3.1.2            | 8                    | 2.12                     | 8.\*, 9.\*                    |
+| 4.4.1        | 3.1.1 - 3.1.2            | 8                    | 2.12                     | 8.\*, 9.\*                    |
+| 4.4.0        | 3.1.1 - 3.1.2            | 8                    | 2.12                     | 8.\*, 9.\*                    |
+| 4.3.1        | 3.1.1 - 3.1.2            | 8                    | 2.12                     | 8.\*, 9.\*                    |
+| 4.3.0        | 3.1.1 - 3.1.2            | 8                    | 2.12                     | 8.\*, 9.\*                    |
+| 4.2.0        | 3.1.1                    | 8                    | 2.12                     | 8.\*                          |
+| 4.1.0        | 3.1.1                    | 8                    | 2.12                     | 8.\*                          |
+| 4.0.0        | 3.1.1                    | 8                    | 2.12                     | 8.\*                          |
+| 4.0.0-beta.3 | 3.1.1                    | 8                    | 2.12                     | 8.\*                          |
+| 4.0.0-beta.2 | 3.1.1                    | 8                    | 2.12                     | 8.\*                          |
+| 4.0.0-beta.1 | 3.1.1                    | 8                    | 2.12                     | 8.\*                          |
+
+### Download
+
+You can use the maven coordinate of the jar to auto install the Spark Connector to your Databricks Runtime 8 from Maven:
+`com.azure.cosmos.spark:azure-cosmos-spark_3-2_2-12:4.14.1`
+
+You can also integrate against Cosmos DB Spark Connector in your SBT project:
+```scala
+libraryDependencies += "com.azure.cosmos.spark" % "azure-cosmos-spark_3-3_2-12" % "4.14.1"
+```
```

Comment:
This looks to be using the wrong version for supporting Databricks Runtime 8 as the table above says only `com.azure.cosmos.spark:azure-cosmos-spark_3-1_2-12:4.14.1` supports Runtime 8

Reply:
Yeah, agreed, I think we need to fix this.

Reply:
Fixed.